pepper-apis/src/main/java/org/broadinstitute/ddp/json/admin/participantslookup/ParticipantsLookupResultRowBase.java

```diff
@@ -0,0 +1,75 @@
+package org.broadinstitute.ddp.json.admin.participantslookup;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Base set of data to be fetched during participants lookup.
+ */
+public class ParticipantsLookupResultRowBase {
+
+    @SerializedName("guid")
+    protected String guid;
```

Comment:
Can we also return `legacyAltPid` and `legacyShortId` in the final response? Some studies (like Angio and MBC) have legacy users and it'll be good to return those two pieces of data. These 2 properties should only be set on the participant object, not the proxy. I'll update the API docs to reflect this.

Reply:
Added